src/App.js

```diff
@@ -1,13 +1,72 @@
-import React from 'react';
+import React from "react";
+
+import TodoList from "./components/TodoComponents/TodoList";
+
+import TodoForm from "./components/TodoComponents/TodoForm";
 
 class App extends React.Component {
   // you will need a place to store your state in this component.
   // design `App` to be the parent component of your application.
   // this component is going to take care of state, and any change handlers you need to work with your state
+  constructor() {
+    super();
+    this.state = {
+      todos: [],
+      task: "",
+      completed: false,
+      id: Date.now()
+    };
+  }
+
+  handleChange = e => {
+    this.setState({ task: e.target.value });
+  };
```

Comment:
Goodjob writing a change handler here, however this would only apply to the task field in state to make it a universal `inputChangeHandler` it would need to look like this.
```js
  handleChange = e => {
    this.setState({ [e.target.name]: e.target.value });
  };
```
as well as add a `name` attribute to the input field that references the same exact spelling as the target you are wishing to change in your state like this.

```js
<input value={task} onChange={handleChange} placeholder="...To Do"  name="task" />
```